rand_hc/README.md

```diff
@@ -0,0 +1,44 @@
+# rand_hc
+
+[![Test Status](https://github.com/rust-random/rand/workflows/Tests/badge.svg?event=push)](https://github.com/rust-random/rand/actions)
+[![Latest version](https://img.shields.io/crates/v/rand_hc.svg)](https://crates.io/crates/rand_hc)
+[[![Book](https://img.shields.io/badge/book-master-yellow.svg)](https://rust-random.github.io/book/)
+[![API](https://img.shields.io/badge/api-master-yellow.svg)](https://rust-random.github.io/rand/rand_hc)
+[![API](https://docs.rs/rand_hc/badge.svg)](https://docs.rs/rand_hc)
+[![Minimum rustc version](https://img.shields.io/badge/rustc-1.36+-lightgray.svg)](https://github.com/rust-random/rand#rust-version-requirements)
```

Comment:
According to the RNGs README, the MSRV is 1.32, but according to the tests it's 1.36. Maybe the CI got copied over without checking this? At this point I don't think it matters though we should fix either the README or CI, and this link (wrong repo).

Reply:
I'll try fixing the CI to see whether we can actually support 1.32.

Reply:
`rand_core` requires Rust 1.36, so we effectively cannot support 1.32. I'll just fix the docs.